prow/gerrit/client/client.go

```diff
@@ -602,7 +602,7 @@ func (h *gerritInstanceHandler) QueryChangesForProject(log logrus.FieldLogger, p
 
 	var opt gerrit.QueryChangeOptions
 	opt.Query = append(opt.Query, strings.Join(append(additionalFilters, "project:"+project), "+"))
-	opt.AdditionalFields = []string{"CURRENT_REVISION", "CURRENT_COMMIT", "CURRENT_FILES", "MESSAGES", "LABELS"}
+	opt.AdditionalFields = []string{"ALL_REVISIONS", "CURRENT_COMMIT", "CURRENT_FILES", "MESSAGES", "LABELS"}
```

Comment:
Are we sure it is safe to omit `CURRENT_REVISION`?  I'm worried that omitting this will cause the [`ChangeInfo.CurrentRevision` field](https://pkg.go.dev/github.com/andygrunwald/go-gerrit#ChangeInfo.CurrentRevision) to not be populated.

Reply:
Confirmed with Gerrit team and the `ChangeInfo.CurrentRevision` field will still be populated.